### DISCUSSION TITLE
EMQ X 4.3-rc.4 Released

### DISCUSSION THREAD

**Discussion by Rory-Z** [emqx/emqx#4613] [General]
EMQ X 4.3-rc.4 is released now, it mainly includes the following changes:

### emqx

**Enhancement:**

- Redis sentinel mode supports SSL connection

  Github PR: [emqx#4553](https://github.com/emqx/emqx/pull/4553)

- WebSocket connection supports obtaining real IP and Port

  Github PR: [emqx#4558](https://github.com/emqx/emqx/pull/4558)

- Support cluster metrics for Prometheus

  Github Issue: [emqx#4548](https://github.com/emqx/emqx/pull/4548)
  Github PR: [emqx#4572](https://github.com/emqx/emqx/pull/4572)

**Bug fixes:**

- Fix the issue of inflight in MQTT bridge

  Github Issue: [emqx#3629](https://github.com/emqx/emqx/issues/3629)
  Github PR: [emqx#4513](https://github.com/emqx/emqx/pull/4513), [emqx#4526](https://github.com/emqx/emqx/pull/4526)
  
- Fix the issue that the multi-language extension hook cannot handle the false value returned

  Github PR: [emqx#4542](https://github.com/emqx/emqx/pull/4542)

- Start modules by default to prevent modules from not working properly after the cluster

  Github PR: [emqx#4547](https://github.com/emqx/emqx/pull/4547)

- Fix the issue that the authentication data cannot be imported

  Github PR: [emqx#4582](https://github.com/emqx/emqx/pull/4582), [emqx#4528](https://github.com/emqx/emqx/pull/4528)

- Fix the issue that the authentication data cannot be imported

  Github PR: [emqx#4563](https://github.com/emqx/emqx/pull/4563)

- Fix the issue that the MQTT-SN gateway will discard DISCONNECT packets in sleep mode

  Github Issue: [emqx#4506](https://github.com/emqx/emqx/issues/4506)
  Github PR: [emqx#4515](https://github.com/emqx/emqx/pull/4515)

- Fix the issue that the LwM2M gateway encodes and decodes numbers into unsigned integers

  Github Issue: [emqx#4499](https://github.com/emqx/emqx/issues/4499)
  Github PR: [emqx#4500](https://github.com/emqx/emqx/pull/4500)

- Fix the issue that some HTTP APIs are unavailable

  Github Issue: [emqx#4472](https://github.com/emqx/emqx/issues/4472)
  Github PR: [emqx#4503](https://github.com/emqx/emqx/pull/4503)

<hr /><em>This discussion was created from the release <a href='https://github.com/emqx/emqx/releases/tag/v4.3-rc.4'>EMQ X 4.3-rc.4 Released</a>.</em>

**Comment by matthewGEA** [comment]
@Rory-Z Does this release include https://github.com/emqx/ekka/pull/86?

**Reply by zmstone** [reply]
@matthewGEA 
yes, it is included.
